aim/tests/unit/test_aim_manager.py

```diff
@@ -952,15 +952,16 @@ class TestHostLinkMixin(object):
 
 class TestHostDomainMappingMixin(object):
     resource_class = infra.HostDomainMapping
-    test_identity_attributes = {'host_name': 'host1.example.com'}
+    test_identity_attributes = {'host_name': 'host1.example.com',
+                                'domain_name': 'ostack1',
+                                'domain_type': 'OpenStack'}
     test_required_attributes = {'host_name': 'host1.example.com',
-                                'vmm_domain_name': 'ostack1',
-                                'physical_domain_name': 'physdom1'}
+                                'domain_name': 'ostack1',
+                                'domain_type': 'OpenStack'}
     test_search_attributes = {'host_name': 'host1.example.com'}
-    test_update_attributes = {'vmm_domain_name': 'ostack2',
-                              'physical_domain_name': 'physdom2'}
-    test_default_values = {'vmm_domain_name': '',
-                           'physical_domain_name': ''}
+    test_update_attributes = {}
+    test_default_values = {'domain_name': '',
```

Comment:
I don't see these 2 default values are being set anywhere in the code so wonder how could this UT pass? I think it makes sense to set domain_type default value to openStack but it does not makes sense to set the default value for domain_name as '' though. Because if we do that, then this entry basically becomes useless. So maybe just set the default value for domain_type then.

Reply:
I'm thinking that the default for the domain_name should be our wildcard ("*").